docs/_includes/Running-Flink-on-Alluxio/license-zk.md

```diff
@@ -0,0 +1,3 @@
+```bash
+$ bin/alluxio fs copyFromLocal LICENSE alluxio://zk@zookeeper_hostname1:2181,zookeeper_hostname2:2181,zookeeper_hostname3:2181/LICENSE
```

Comment:
we are moving away from using code snippets because then it's hard to edit the doc (need to jump between many different files). Please inline instead

Reply:
Delete the Running-Flink-on-Alluxio code sinppets and inline them, thanks